ironfish-rust/src/keys/test.rs

```diff
@@ -115,13 +115,26 @@ fn test_hex_conversion() {
 }
 
 #[test]
-fn test_words_spending_key() {
+fn test_to_words() {
     let key = SaplingKey::generate_key();
```

Comment:
If you wanted to make these tests more deterministic, you could do something like this

```rust
let byte_array = [213, 107, ... ]; // copied from below
SaplingKey::new(byte_array)
assert_eq(words, key.spending_key)
```